docs/setup.md

```diff
@@ -24,6 +24,9 @@ protected function bootstrapCli(): void
 
 **We recommend using migrations for managing the schema of your test DB with the [CakePHP Migrator tool.](https://book.cakephp.org/migrations/2/en/index.html#using-migrations-for-tests)**
 
+## CakePHP 4.4+
+For CakePHP 4.4 or higher applications, add the `TruncateDirtyTables` trait to the top of your test classes to automatically truncate dirty tables after each test runs.
+No modification to your PHPUnit configuration is required when using the trait.

```

Comment:
as soon as #181 is merged I will rebase this branch and this will go away

Reply:
Note that this should read CakePHP ≥ 4.3

Reply:
Updates made to file in other PR